chapter02/2.5 - Sum Lists/sumLists - forward.js

```diff
@@ -0,0 +1,83 @@
+const LinkedList = require('./../util/LinkedList')
+const printList = require('./../util/printList')
+
+function sumLinkedListsForward(list1, list2) {
+  if (!list1 && !list2) {
```

Comment:
I believe the if statement is correct. The purpose of this line is to return null if no Lists are entered. If one List is entered it will just return that list (because x + 0 = x)

so if
```
const list1 = new LinkedList(6)
const list2 = null
console.log(!list1 && !list2 ) //false

const list1 = null
const list2 = new LinkedList(3)
console.log(!list1 && !list2 ) //false

const list1 = null
const list2 = null
console.log(!list1 && !list2) //true
```

Reply:
I see, ok, makes sense.